docs/tutorials/nb_imputation_comparison.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Imputation Comparison\n",
+    "\n",
+    "[![open in colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ing-bank/probatus/blob/master/docs/tutorials/nb_imputation_comparison.ipynb)"
```

Comment:
When you finish,  please make sure that this link works, and you have the following cell:
```
%%capture
!pip install probatus
!pip install lightgbm
```

This installs probatus and lightgbm in colab environment

Reply:
Sure. I think once the note is in the master branch. I can do the check. As of now I have left the pip installation commands as comments, which the users can activate in the collab environment.